infrastructure/template.yaml

```diff
@@ -98,6 +98,37 @@ Conditions:
 
 Globals:
   Function:
+    Environment:
+      Variables:
+        AWS_LAMBDA_EXEC_WRAPPER: /opt/dynatrace
+        DT_CONNECTION_AUTH_TOKEN: !Sub
+          - '{{resolve:secretsmanager:${SecretArn}:SecretString:DT_CONNECTION_AUTH_TOKEN}}'
+          - SecretArn: !FindInMap [ EnvironmentConfiguration, !Ref Environment, dynatraceSecretArn ]
+        DT_CONNECTION_BASE_URL: !Sub
+          - '{{resolve:secretsmanager:${SecretArn}:SecretString:DT_CONNECTION_BASE_URL}}'
+          - SecretArn: !FindInMap [ EnvironmentConfiguration, !Ref Environment, dynatraceSecretArn ]
+        DT_CLUSTER_ID: !Sub
+          - '{{resolve:secretsmanager:${SecretArn}:SecretString:DT_CLUSTER_ID}}'
+          - SecretArn: !FindInMap [ EnvironmentConfiguration, !Ref Environment, dynatraceSecretArn ]
+        DT_LOG_COLLECTION_AUTH_TOKEN: !Sub
+          - '{{resolve:secretsmanager:${SecretArn}:SecretString:DT_LOG_COLLECTION_AUTH_TOKEN}}'
+          - SecretArn: !FindInMap [ EnvironmentConfiguration, !Ref Environment, dynatraceSecretArn ]
+        DT_TENANT: !Sub
+          - '{{resolve:secretsmanager:${SecretArn}:SecretString:DT_TENANT}}'
+          - SecretArn: !FindInMap [ EnvironmentConfiguration, !Ref Environment, dynatraceSecretArn ]
+        DT_OPEN_TELEMETRY_ENABLE_INTEGRATION: "true"
+    Architectures: ["arm64"]
+    MemorySize: 128
+    KmsKeyArn: !GetAtt LambdaKMSKey.Arn
+    ReservedConcurrentExecutions: 10
+    Runtime: nodejs18.x
+    Timeout: 5
+    VpcConfig:
+      SubnetIds:
+          - Fn::ImportValue: !Sub "${VpcStackName}-ProtectedSubnetIdA"
+          - Fn::ImportValue: !Sub "${VpcStackName}-ProtectedSubnetIdB"
```

Comment:
Just to confirm are deployments only set across two available zones? I know there's an option for three. You should be able to confirm the number of AZs available based on the CloudFormation VPC AvailabilityZoneCount parameter. Not necessarily in scope of Dynatrace work but just worth noting.

Reply:
Yes, its set to 2 in dev, build, integration and staging.  I've raised https://govukverify.atlassian.net/browse/OLH-1281 to consider using all three AZs/

Reply:
That's right. The first version of the VPC stack only exported two subnets which is what we originally set up on. At somepoint between then and now they've added the option for the third AZ